http://bugs.debian.org/cgi-bin/bugreport.cgi?msg=5;bug=400185

There are some errors in gssapi.h. #if is used instead of #if defined()

Julien Danjou <acid@debian.org>

--- a/src/gssglue/gssapi/gssapi.h.in	2006-04-09 06:27:24.000000000 +1000
+++ b/src/gssglue/gssapi/gssapi.h.in	2009-06-06 19:08:15.000000000 +1000
@@ -51,7 +51,7 @@
 extern "C" {
 #endif /* __cplusplus */
 
-#if TARGET_OS_MAC
+#if defined(TARGET_OS_MAC)
     #if defined(__MWERKS__)
         #pragma import on
         #pragma enumsalwaysint on
@@ -107,7 +107,7 @@ extern "C" {
 /*
  * If the platform supports the xom.h header file, it should be included here.
  */
-#if	HAVE_XOM_H
+#if	defined(HAVE_XOM_H)
 #include <xom.h>
 #endif	/* HAVE_XOM_H */
 
@@ -826,7 +826,7 @@ PROTOTYPE(	(OM_uint32  *,		/* minor_stat
 		void *			/* ktypes */
 	));
 
-#if TARGET_OS_MAC
+#if defined(TARGET_OS_MAC)
     #if defined(__MWERKS__)
         #pragma enumsalwaysint reset
         #pragma import reset
